packages/manager/modules/pci/src/projects/project/kubernetes/details/audit-logs/translations/Messages_fr_FR.json

```diff
@@ -3,5 +3,5 @@
   "kube_logs_list_tab_title": "Liste",
   "kube_logs_description": "En transférant les logs d'audit de votre cluster Kubernetes dans Logs Data Platform, vous pouvez les stocker, effectuer des recherches dans vos logs, définir des alertes et construire des tableaux de bord.",
   "logs_list_title": "Service K8S",
-  "logs_list_description": "En transférant les audit logs de votre cluster Kubernetes dans Logs Data Platform vous pouvez les centraliser depuis d'autres sources, gérer leur rétention, effectuer des recherches, définir des alertes ainsi que des tableaux de bord pour observer facilement vos clusters."
+  "logs_list_description_audit": "En transférant les audit logs de votre cluster Kubernetes dans Logs Data Platform vous pouvez les centraliser depuis d'autres sources, gérer leur rétention, effectuer des recherches, définir des alertes ainsi que des tableaux de bord pour observer facilement vos clusters."
```

Comment:
I see the API is returning (at least one) more log kind, shouldn't you add a matching translation key ?

Reply:
The other value currently returned ("beta") is a mock one temporarily available only for QA